src/main/features/claim/guards/onlyDefendantCanDoItGuard.ts

```diff
@@ -0,0 +1,31 @@
+import * as express from 'express'
+
+import { GuardFactory } from 'response/guards/guardFactory'
+import { Paths } from 'response/paths'
+
+import { Claim } from 'claims/models/claim'
+import { UUIDUtils } from 'shared/utils/uuidUtils'
+
+export class OnlyDefendantCanDoItGuard {
+  /**
+   * Makes sure that current user is linked as a defendant to current claim.
+   *
+   * @returns {express.RequestHandler} - request handler middleware
+   */
+  static requestHandler (): express.RequestHandler {
+    return GuardFactory.create((res: express.Response) => {
+
+      const claim: Claim = res.locals.claim
+      const user: User = res.locals.user
+
+      if (!claim || !claim.defendantId || !user) {
+        return false
+      }
+
+      return claim.defendantId === user.id
+    }, (req: express.Request, res: express.Response): void => {
+      const externalId: string = UUIDUtils.extractFrom(req.path)
+      res.redirect(Paths.claimDetailsPage.evaluateUri({ externalId: externalId }))
```

Comment:
ok

Reply:
done

Reply:
Sorry maybe I wasn't clear enough, we do not want to have both `IsDefendantInCaseGuard` and `OnlyDefendantCanDoItGuard`. I think it makes sense to use `IsDefendantInCaseGuard` since it is more feature rich.